### PR TITLE
[WASM] Add PyDocument and PyElement classes to browser

### PR DESCRIPTION
**Pull request by coolreader18**
I was going to do `.on` for events, but I realized that it would require (a) a different class for every single possible Event subclass, or (b) a `PyJsValue` class with `__getattr__`, which I'm going to do but in a separate PR.

I also fixed that issue from before with importing the `json` module - `builtins.__import__` tried to get the current frame, and if there wasn't anything on the stack it just failed because `vm.current_frame()` used `frames[idx]` which panics instead of `.get(idx)` which returns a result in the case of an invalid index.

### CODE REVIEW

File: wasm/lib/src/browser_module.rs [hunk 1]
```diff
@@ -270,6 +263,65 @@ fn promise_catch(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
     Ok(PyPromise::new_obj(promise_type, ret_promise))
 }
 
+#[derive(Debug)]
+struct PyDocument {
```
Comment (OddCoincidence) [inline wasm/lib/src/browser_module.rs]: nit: can this just be `Document` (and same for others)? Outside of the core built-in types, the prefixes just seem like noise to me.

File: wasm/lib/src/browser_module.rs [hunk 2]
```diff
@@ -270,6 +263,65 @@ fn promise_catch(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
...
+}
+type PyElementRef = PyRef<PyElement>;
+
+impl PyValue for PyElement {
+    fn class(vm: &VirtualMachine) -> PyObjectRef {
+        vm.class("browser", "Element")
+    }
+}
+
+fn elem_get_attr(
```
Comment (OddCoincidence) [inline wasm/lib/src/browser_module.rs]: I know it won't let you do `impl PyElementRef`, but how about `impl PyElement`? In the future we'll be able to make it follow the pattern once `arbitrary-self-types` are allowed.

Comment (coolreader18) [reply wasm/lib/src/browser_module.rs]: Oh, I didn't consider that. Sounds good!

**Comment by OddCoincidence** [comment]
> I was going to do .on for events, but I realized that it would require (a) a different class for every single possible Event subclass, or (b) a PyJsValue class with __getattr__, which I'm going to do but in a separate PR.

The [`gloo`](https://github.com/rustwasm/gloo) project is working on a more idiomatic and type safe wrapper for events, so it might be worth waiting to see if we can base it off of that.

File: wasm/lib/src/browser_module.rs (hunk 1 above)
Comment (coolreader18) [reply wasm/lib/src/browser_module.rs]: Sure, for `PyJsValue` I think I'll keep the prefix but for these I'll drop it.

**Comment by coolreader18** [comment]
It'd still require (probably manually) making Python classes for each possible `Event` struct, so I think it's easiest to just go with a `PyJsValue` for now.

**Comment by coolreader18** [comment]
I see what you mean, using `gloo_events` for Python event listeners. That sounds like a good idea, so I'll wait for that.

**Pull request merged**